core/templates/pages/topic-editor-page/subtopic-editor/subtopic-editor-tab.component.ts

```diff
@@ -186,6 +187,27 @@ angular.module('oppia').component('subtopicEditorTab', {
         ctrl.skillsListIsShown = !ctrl.skillsListIsShown;
       };
 
+      ctrl.showSkillEditOptions = function(index) {
+        ctrl.selectedSkillEditOptionsIndex = (
+            (ctrl.selectedSkillEditOptionsIndex === index) ? null : index);
```

Comment:
Let's use either null or -1 to "unset" selectedSkillEditOptionsIndex and not both (we seem to use -1 in line 196 below).

Reply:
Yeah -1 seems better. Done